Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -154,6 +154,7 @@ private static TDriverOptions SetCommonOptions<TDriverOptions>(this TDriverOptio
         {
             var fakeCameraSourceFilePath = configuration.FakeVideoSource.SaveVideoToTempFolder();
 
+            options.AddArgument("disable-gesture-requirement-for-media-playback");
```

Comment:
I added this, so a video would start no matter what (just as now in the BehaviorVueTests). There could be cases where it wouldn't by default.

Reply:
Could you add the explanation as a comment in the code instead?